nimdoc/testproject/expected/subdir/subdir_b/utils.html

```diff
@@ -206,9 +206,9 @@ <h1><a class="toc-backref" href="#18">Templates</a></h1>
 <dt><pre><span class="Keyword">template</span> <a href="#fromUtilsGen.t"><span class="Identifier">fromUtilsGen</span></a><span class="Other">(</span><span class="Other">)</span><span class="Other">:</span> <span class="Identifier">untyped</span></pre></dt>
 <dd>
 
-this should be shown in utils.html
+should be shown in utils.html only
 <p><strong class="examples_text">Example:</strong></p>
-<pre class="listing"><span class="Identifier">assert</span> <span class="DecNumber">3</span><span class="Operator">*</span><span class="DecNumber">2</span> <span class="Operator">==</span> <span class="DecNumber">6</span></pre>ditto
+<pre class="listing"><span class="Keyword">discard</span> <span class="StringLit">&quot;should be in utils.html only, not in module that calls fromUtilsGen&quot;</span></pre>ditto
```

Comment:
This seems to be worse, now the runnableExample for template `formUtilsGen` disappeared?

Reply:
no it's there, see `hould be in utils.html only, not in module that calls fromUtilsGen`